nft/collection/collection.go

```diff
@@ -60,26 +46,24 @@ var (
 
 type CollectionPolicy struct {
```

Comment:
CollectionPolicy is used outside of package with collection.CollectionPolicy. This is a case where the same name is used repeatedly, so it would be better to just use the type name as Policy.